pkg/operators/common/kubeinventorycache.go

```diff
@@ -29,30 +30,106 @@ import (
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/k8sutil"
 )
 
+type SlimObjectMeta struct {
+	Name            string                  `json:"name"`
+	Namespace       string                  `json:"namespace"`
+	ResourceVersion string                  `json:"resourceVersion"`
+	Labels          map[string]string       `json:"labels"`
+	OwnerReferences []metav1.OwnerReference `json:"ownerReferences"`
```

Comment:
Are these included in this PR to be potentially useful for other code parts? AFAIR they are not used in K8sInvCache

Reply:
It's used by the kubenameresolver: https://github.com/inspektor-gadget/inspektor-gadget/blob/74dd7ae3ff2921ac74af158810ae2fc70c487107/pkg/operators/kubenameresolver/kubenameresolver.go#L122